Privacy-policy-snippets.md

```diff
@@ -0,0 +1,36 @@
+# Privacy Policy Snippets
+
+Cut and paste-able snippets that describe how core collects, manages and shares data provided by site users and visitors.
```

Comment:
Should that be "Copy and paste-able.."?

Reply:
Fixed in https://github.com/gdpr-compliance/info/pull/8/commits/3481f3b64f5138e52a3fbfc2c06c6d6c9c14859b